boa3/internal/cli_commands/build_command.py

```diff
@@ -26,6 +27,12 @@ def add_arguments_and_callback(self):
         self.parser.add_argument("-e", "--env",
                                  type=str,
                                  help="Set the contract environment for compiling.")
+        self.parser.add_argument("-op", "--output-path",
+                                 type=str,
+                                 default=None,
+                                 help="Chooses the name and where the compiled files will be generated, "
+                                      "if not specified it will be generated on the same directory with the same name "
+                                      "as the python file.")
```

Comment:
```suggestion
        self.parser.add_argument("-o", "--output",
                                 metavar='NEF_OUTPUT',
                                 type=str,
                                 default=None,
                                 help="Chooses the name and where the compiled files will be generated, "
                                      "if not specified it will be generated on the same directory with the same name "
                                      "as the python file.")
```
cli applications usually use "-o" to set their outputs, we should follow this
also the help message should specify that it's expected a .nef file path